reflex/state.py

```diff
@@ -1532,14 +1539,22 @@ async def _process_event(
                 yield state._as_state_update(handler, events, final=True)
 
         # If an error occurs, throw a window alert.
-        except Exception:
-            error = traceback.format_exc()
-            print(error)
-            yield state._as_state_update(
-                handler,
-                window_alert("An error occurred. See logs for details."),
-                final=True,
-            )
+        except Exception as e:
+            stack_trace = traceback.format_exc()
+
+            config = get_config()
+
+            # Call the custom backend exception handler if specified. If not, fallback to the default handler.
+            if config.backend_exception_handler:
+                config.backend_exception_handler(message=str(e), stack=stack_trace)
+            else:
+                default_backend_exception_handler(message=str(e), stack=stack_trace)
+
+                yield state._as_state_update(
```

Comment:
shouldn't this happen in `default_backend_exception_handler`?

Reply:
yes, I have just made changes that should fully preserve the default behaviour